mock-vm/backend/src/contract_env.rs

```diff
@@ -124,3 +126,11 @@ pub fn verify_signature(message: &Bytes, signature: &Bytes, public_key: &PublicK
     let mock_signature_bytes = Bytes::from(message);
     mock_signature_bytes == *signature
 }
+
+/// Creates a hash of the given input. Uses default hash for given backend.
+pub fn hash<T: AsRef<[u8]>>(input: T) -> Bytes {
```

Comment:
I wouldn't wrap the bytes. Most likely, a user will use the result in some further processing, so having raw bytes is a more flexible approach.

Reply:
Fixed